galite-tests/src/test/kotlin/org/kopi/galite/tests/dsl/ReportDSLTests.kt

```diff
@@ -73,9 +73,10 @@ class ReportDSLTests : VApplicationTestBase() {
   @Test
   fun `test report rows`() {
     val report = ReportWithData()
-    val reportModel = report.model
+    report.model.initReport()
+    val reportModel = report.model.model
 
-    assertEquals(3, report.reportRows.size)
+    assertEquals(4, reportModel.getVisibleRowCount())
```

Comment:
remove it because you check it bellow or check columns, 
do we need to add data in this test?

Reply:
Removed the duplicated check and now we are verifying if the added record exists in the model.